packages/image_picker/image_picker_windows/lib/image_picker_windows.dart

```diff
@@ -38,10 +38,11 @@ class ImagePickerWindows extends ImagePickerPlatform {
   /// Constructs a ImagePickerWindows.
   ImagePickerWindows();
 
+  static FileSelectorWindows get _fileSelectorInstance => FileSelectorWindows();
```

Comment:
This doesn't do what I expect you intended; this is basically just an alias for the constructor. Assuming you wanted a shared instance you want:
```
static late FileSelectorWindows = FileSelectorWindows();
```

Reply:
Oh, my bad. I want a simple singleton, and I don't see many reasons to late initialize this. I believe this is fine:
```dart
  static final FileSelectorWindows _fileSelectorInstance = FileSelectorWindows();
```